contributors/triage/guideline.md

```diff
@@ -0,0 +1,62 @@
+
+## Using the bot
+
+Most people do not have the ability to interact with the github issues and labels, for that we use a bot to manage labelling and triaging. The bot has a set of [commands and permissions](https://github.com/kubernetes/test-infra/blob/master/commands.md) and this document will cover the basic ones.
+
+## Determine if it’s a support request
+
+Sometimes users ask for support requests in issues; these are usually support requests for people who need help configuring some aspect of Kubernetes. These should be directed to our [support structures](https://github.com/kubernetes/community/blob/master/contributors/devel/on-call-user-support.md) and then closed. Also, if the issue is clearly abandoned or in the wrong place, it should be closed. Keep in mind that only issue reporter, assignees and component organization members can `/close` issue. If you do not have such privilege, just comment your findings.
+
+## Find the right SIG(s)
+Components are divided among [Special Interest Groups (SIGs)](https://github.com/kubernetes/community/blob/master/sig-list.md). Find a proper SIG for the ownership of the issue using the bot:
+
+* Typing `/sig network` in a comment should add the sig/network label, for example.
+* Multiword SIGs use dashes, for example `/sig cluster-lifecycle`.
+
+## Validate if the issue is real
+
+Validate if the problem is real by reproducing it. If reproducible, move to the next step of defining priority. You may need to contact the issue reporter in the following cases:
+* Do a quick duplicate search to see if the issue has been reported already. If a duplicate is found, let the issue reporter know it by marking it duplicate.
+* If you can not reproduce the issue, contact the issue reporter with your findings and close the issue if both the parties agree that it could not be reproduced.
+* If the issue is non-trivial to reproduce, work with issue reporter and let SIG know of your findings.
+* If you do not get a response in 20 days, then close the issue with appropriate comment.
+
+## Define priority
+After you validate the issue, add a comment of what you think of priority of fixing issue from priority labels provided below. The owner from appropriate SIG will correct the label if needed.
+* `priority/critical-urgent` or `priority/failing-test` - Critical. Fix it now.
+* `priority/important-soon` - High priority. Fix it as soon as possible.
+* `priority/important-longterm` - High priority over the long duration.
+* `priority/backlog` - Medium priority.
+* `priority/awaiting-more-evidence` - Low priority.
+
+To read priorities in detail, please refer to the [Priorities](https://github.com/kubernetes/community/blob/master/contributors/devel/issues.md#priorities).
+
+## Set ownership
```

Comment:
Everything to this point has been a logical progression. Ideally you could reference a single logic flow chart that a bot could link and reference such that new contributors know what state the issue is in.